android/app/src/main/java/updated/mysterium/vpn/common/extensions/IntExtensions.kt

```diff
@@ -0,0 +1,6 @@
+package updated.mysterium.vpn.common.extensions
+
+import android.content.res.Resources
+
+val Int.px: Int
```

Comment:
It's better create util class `DimenUtils` and add method there `dpToPx`, so it will be more clear usage, because this case is too convenient.